README.md

```diff
@@ -1,13 +1,78 @@
+
+#### How do I get a film from a list?
+Enter the username of the list creator, followed by a <kbd>/</kbd> and the list title. For example, you would search through [this list](https://letterboxd.com/jack/list/its-someones-favorite-movie/), by entering: `jack/its-someones-favorite-movie` ([see it in action on Watchlist Picker](https://watchlistpicker.com/?u=jack/its-someones-favorite-movie)). If you want to just copy the text straight out of the URL, `jack/list/its-someones-favorite-movie` will work too.
+	
+#### How do I search multiple lists at once?
+Enter all of the lists and usernames into the search bar, seperated by a space or comma.
+
+#### How do I only show films that are on all of the lists?
+By default, the lists are combined into a giant pool of movies. To get only movies that appear on all of the lists you've entered (the intersection), click on "Advanced Search" (located right below the search bar), and change "Union" to "Intersection".
+
+#### Why does it say the intersection between the lists is empty?
+There was not a single movie that appeared on every single one of the lists you entered. If you are certain there should be, make sure all of the lists and usernames are spelt correctly.
+
+#### Why does it keep loading a new movie when I refresh the page?
+The URL of the page contains your search criteria, so you can easily bookmark your searches or share with friends. To reset it, simply click on the logo.
+
+#### How do I turn on dark-mode?
+Click on the toggle switch in the top right corner.
+
+#### How do I turn off dark-mode?
+Click on the toggle switch in the top right corner.
+
+#### Why is the logo spinning?
+It's like the Letterboxd logo, but a slot machine.
+
+#### Why is the loading bar stuck?
+The site may have lost connection to the API, or it's just taking longer to search through the list than anticipated (often the really big lists can take a minute or two). Try opening the site again in a new tab.
+
+#### How can I support the site?
+If you wish to help cover operational costs, we have a [ko-fi](https://ko-fi.com/goodbyte).
+
+#### I'm having trouble using the site.
+If at any point you have issues using the website, please do not hesitate to contact us at [support@goodbyte.ca](mailto:support@goodbyte.ca).
+
+
+## Operational notes
+
+[[Building locally](#building-locally)] [[So where are things?](#so-where-are-things)]
+
+> Note: please do **not** push any commits to the `main` branch. Instead, create a new branch, build it locally, and then create a pull request. Once the PR has been tested in Preview (and approved by another member) you can merge it into `main`.
+
+### Building locally
+
+[ TODO ]
```

Comment:
This is for you to add @HoloPollock, as I mostly just edit in Github.

Reply:
Done